src/loaders/zarr_utils/ChunkPrefetchIterator.ts

```diff
@@ -61,11 +70,26 @@ export default class ChunkPrefetchIterator {
 
     for (const [direction, start] of extrema.flat().entries()) {
       const dimension = direction >> 1;
-      let end: number;
+      const tczyxIndex = dimension + Number(dimension !== 0);
```

Comment:
I don't quite understand this line. What's the `Number(dimension !== 0)` for? I trust that it's important, I just can't figure out why....

Reply:
Yeah - it's a silly, hard-to-parse trick to add 0 when `dimension` is 0 and 1 otherwise, to skip index 1. The contents of the function you read are inlined here just because I wanted that intermediate `dimension` value.